### PR TITLE
Allow API keys to download from library (for BAPSicle)

### PR DESCRIPTION
**Pull request by mstratford**
Todo:
- [x] Check that new setups will follow this (probably need to set the schema jsons).

### CODE REVIEW

**Comment by URY-GitHub** [comment]
Thank you for this pull request, @mstratford!

@SplashyFish14, would you mind reviewing this PR, please? Feel free to ask anyone for help if you're not quite sure what you're doing.

**Comment by mstratford** [comment]
@markspolakovs This is ready now.

File: schema/data-actionsauth.json [hunk 1]
```diff
@@ -81,10 +81,8 @@
     ["NIPSWeb", "live", "AUTH_USENWLIVE"],
     ["NIPSWeb", "load_auto_managed", "AUTH_USENIPSWEB"],
     ["NIPSWeb", "load_aux_lib", "AUTH_USENIPSWEB"],
-    ["NIPSWeb", "managed_play", "AUTH_USENIPSWEB"],
     ["NIPSWeb", "manage_library", "AUTH_USENIPSWEB"],
     ["NIPSWeb", "recv_ops", "AUTH_USENIPSWEB"],
-    ["NIPSWeb", "secure_play", "AUTH_USENIPSWEB"],
```
Comment (markspolakovs) [inline schema/data-actionsauth.json:87]: Shouldn't this need a `GLOBAL_ACCESS` to replace it? I'd imagine this hits the GUI MyRadio login check code

**Review by markspolakovs — APPROVED** [review]
Looks good modulo comment

File: schema/data-actionsauth.json (hunk 1 above)
Comment (mstratford) [reply schema/data-actionsauth.json:87]: Woops, yes.

**Pull request merged**